examples/cloudflare-worker-loader/src/env.ts

```diff
@@ -0,0 +1,11 @@
+import type { worker } from "../alchemy.run.js";
```

Comment:
shouldn't this be `../alchemy.run.ts`

Reply:
I actually copied from the cloudflare-worker example - it turns out the examples use js/ts inconsistently (also `env.ts`/`env.d.ts`).

Reply:
Since I'd grepped already - https://github.com/alchemy-run/alchemy/pull/1068